README.md

```diff
@@ -151,6 +151,14 @@ Access the `user$` observable on the `AuthService` instance to retrieve the user
 </ul>
 ```
 
+### Handle errors
+
+Errors in the login flow can be captured by subscribing to the `error$` observable:
+
```

Comment:
Is there any error reference we could link to? e.g. spa js might have a list of errors

Reply:
We're just forwarding errors that come from the server, usually. SPA SDK doesn't have a list of such errors.